### DISCUSSION TITLE
泵bug

### DISCUSSION THREAD

**Discussion by zgybd** [IndustrialCraftMC/IndustrialCraft2#19] [【漏洞反馈】Bugs Report]
1.泵在没电状态下，在上方放水，水会无限抽搐
<img width="1280" alt="04dcf27fd66598f15d42b8130a0f97b" src="https://user-images.githubusercontent.com/69385074/236503314-a77bb161-dce7-4665-a624-6e3bf1aa2fab.png">
2.泵在有电的状态，可以无线吸一格水，直到装满自己
<img width="1280" alt="f123b931433c95a3944ac849a06c299" src="https://user-images.githubusercontent.com/69385074/236504554-f69fb50e-4c01-4af0-a2e2-2c52ee89ad8e.png">

**Comment by AbbottMc** [comment]
我这边好像没法重现，你能给点具体的重现方式吗？

**Reply by zgybd** [reply]
第一种情况

https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/69385074/1710b5b9-57e7-4908-b2d7-392cd9d7b259

**Reply by zgybd** [reply]
**第二种情况**

https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/69385074/4b3d56f4-4448-4813-b365-ca1bc5152719

**Reply by AbbottMc** [reply]
ok，收到，我这边测试一下看看

**Comment by AbbottMc** [comment]
这个是forge端的bug，fabric端没有类似的情况，后续会尝试修复
